libs/gpu-dawn/sdk.zig

```diff
@@ -564,14 +568,11 @@ pub fn Sdk(comptime deps: anytype) type {
                 break :blk separate_lib;
             };
 
-            // TODO(build-system): pass system SDK options through
-            try deps.glfw.link(b, lib, .{ .system_sdk = .{ .set_sysroot = false } });
-
             var cpp_flags = std.ArrayList([]const u8).init(b.allocator);
             try appendFlags(step, &cpp_flags, options.debug, true);
             try appendDawnEnableBackendTypeFlags(&cpp_flags, options);
             try cpp_flags.appendSlice(&.{
-                include(deps.glfw_include_dir),
+                "-I" ++ deps.glfw_include_dir,
```

Comment:
Why like this instead of using `include()`?

Reply:
because `include()` make it relative to `/libs/gpu-dawn/`. so you can't e.g give it `/libs/glfw/upstream/glfw/include` from `/build.zig`